### PR TITLE
patron: improve patron detail view

### PR DESCRIPTION
**Pull request by zannkukai**
* Improves patron detail view.

Co-authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 1]
```diff
@@ -16,247 +16,168 @@
...
-        </dt>
-        <dd class="col-sm-7 col-md-8 mb-0">
-          <ul class="pl-0">
-            <li *ngFor="let library of patron.libraries" class="list-unstyled">
+          <dt class="col-1 text-right"><i class="fa fa-home"></i></dt>
+          <dt class="col-3" translate>Address</dt>
+          <dd class="col-8">
+            {{ patron.street }}<br/>
+            {{ patron.postal_code }} {{ patron.city }}<br/>
+            {{ 'country_' + patron.country | translate}}
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:64]: Missing space after `translate`

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 2]
```diff
@@ -16,247 +16,168 @@
...
+      </div>
+    </ng-container>
+    <!-- USER INFORMATIONS SECTION ============================================= -->
+    <div class="card-header" translate>
+      <button type="button" class="btn-shore-more"
+              [ngClass]="{'btn-expanded': !sectionCollapsed.user, 'btn-collapsed': sectionCollapsed.user}"
+              (click)="sectionCollapsed.user = !sectionCollapsed.user"
+              [attr.aria-expanded]="!sectionCollapsed.user" aria-controls="collapse">
+      </button>
+      User informations
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:94]: Translate ?

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 3]
```diff
@@ -16,247 +16,168 @@
...
+    </dl>
+    <!-- PATRON INFORMATIONS SECTION =========================================== -->
+    <ng-container *ngIf="patron.patron">
+      <div class="card-header" translate>
+        <button type="button" class="btn-shore-more"
+                [ngClass]="{'btn-expanded': !sectionCollapsed.patron, 'btn-collapsed': sectionCollapsed.patron}"
+                (click)="sectionCollapsed.patron = !sectionCollapsed.patron"
+                [attr.aria-expanded]="!sectionCollapsed.patron" aria-controls="collapse">
+        </button>
+        Patron Information
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:120]: Translate ?

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 4]
```diff
@@ -16,247 +16,168 @@
...
+      </div>
+      <dl class="card-body row" [hidden]="sectionCollapsed.patron">
+        <dt class="col-1 text-right"><i class="fa fa-barcode"></i></dt>
+        <dt class="col-3" translate>Barcode</dt>
+        <dd class="col-8">
+          <a [routerLink]="['/circulation', 'patron', patron.patron.barcode[0]]">{{ patron.patron.barcode | join: ', '}}</a>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:126]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 5]
```diff
@@ -16,247 +16,168 @@
...
+        </ng-container>
+        <ng-container *ngIf="patron.local_codes">
+          <dt class="col-3 offset-1" translate>Local code</dt>
+          <dd class="col-8">{{ patron.local_codes | join: ',' }}</dd>
+        </ng-container>
+        <ng-container *ngIf="patron.second_address">
+          <dt class="col-1 text-right"><i class="fa fa-home"></i> </dt>
+          <dt class="col-3" translate>Second address</dt>
+          <dd class="col-8">
+            {{ patron.second_address.street  }}<br/>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:153]: extra space

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 6]
```diff
@@ -16,247 +16,168 @@
...
+            {{ patron.second_address.postal_code }} {{ patron.second_address.city }}<br/>
+            {{ 'country_' + patron.second_address.country | translate}}
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:155]: Missing space before `}`

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 7]
```diff
@@ -16,247 +16,168 @@
...
+    </ng-container>
+    <!-- LIBRARIAN INFORMATIONS SECTION ======================================== -->
+    <ng-container *ngIf="isLibrarian">
+      <div class="card-header" translate>
+        <button type="button" class="btn-shore-more"
+                [ngClass]="{'btn-expanded': !sectionCollapsed.librarian, 'btn-collapsed': sectionCollapsed.librarian}"
+                (click)="sectionCollapsed.librarian = !sectionCollapsed.librarian"
+                [attr.aria-expanded]="!sectionCollapsed.librarian" aria-controls="collapse">
+        </button>
+        Librarian Information
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html:168]: Translate ?

**Review by Garfield-fr — APPROVED** [review]

File: projects/admin/src/app/scss/styles.scss [hunk 8]
```diff
@@ -122,6 +122,29 @@ json-schema-form {
   vertical-align: top;
 }
 
+
+/* SHOW MORE BUTTON ====================================================== */
+.btn-shore-more {
+  border: 1px solid $border-color;
```
[the remark this answers is not in the document]
Comment (zannkukai) [reply projects/admin/src/app/scss/styles.scss:128]: Done with`$border-width`.
For border radius, it depends of the browser relative size. If user choose to zoom, the radius need to be the double of the font-size. So `2rem` seems the good solution.

**Pull request merged**